text/0000-lifecycle-prelease-version-and-experimental-section.md

```diff
@@ -31,65 +31,41 @@ This proposes two changes to Lifecycle:
 * `experimental` is a new API mode for testing implementation of the API similar to the modes defined in [RFC#49](https://github.com/buildpacks/rfcs/blob/main/text/0049-multi-api-lifecycle-descriptor.md).
 
 ## Prerelease API
-
 A prerelease API is a non-finalized API but is in a testable state. The the API version must be of the format: `<major>.<minor>-<alphanumeric>`, i.e. `1.0-alpha1`. Though these aren't final, there will be a tagged and release version of the appropriate spec for it. This way users can see what is supported when it's used. In most cases, these will be cut from the corresponding branch in `buildpacks/spec`, i.e. Buildpack API `0.5-alpha1` would be cut from the `buildpack/0.5` branch.
 
 When Lifecycle supports a prerelease API, it can be treated like any other API version and be used in any mode: `experimental`, `supported`, `deprecated`.
 
-## Experimental API Mode
-
-This is a new mode for testing implementation of a API. The API can be a full release or a prerelease API.
-
-New `CNB_PLATFORM_EXPERIMENTAL_MODE` and `CNB_BUILDPACK_EXPERIMENTAL` mode environment variables will control experimental mode with:
-allowed values: `warn`, `error`, `silent`
-default value: `warn`
-
-**When** the `CNB_PLATFROM_API` environment variable is set to an API version in the deprecated platform API, the lifecycle shall:
- - **If** `CNB_PLATFORM_EXPERIMENTAL_MODE` is unset, **Then** print a warning and continue
- - **If** `CNB_PLATFORM_EXPERIMENTAL_MODE=warn`, **Then** print a warning and continue
- - **If** `CNB_PLATFORM_EXPERIMENTAL_MODE=error`, **Then** fail
- - **If** `CNB_PLATFORM_EXPERIMENTAL_MODE=silent`, **Then** continue w/o warning
-
-**When** the `api` field in a `buildpack.toml` file is set to an API version in the deprecated buildpack API range the lifecycle shall:
- - **If** `CNB_BUILDPACK_EXPERIMENTAL_MODE` is unset, **Then** print a warning and continue
- - **If** `CNB_BUILDPACK_EXPERIMENTAL_MODE=warn`, **Then** print a warning and continue
- - **If** `CNB_BUILDPACK_EXPERIMENTAL_MODE=error`, **Then** fail
- - **If** `CNB_BUILDPACK_EXPERIMENTAL_MODE=silent`, **Then** continue w/o warning
+### Experimental Section in the API
```

Comment:
I think we probably want to require users to explicitly opt into experimental features

We could do this with a boolean `CNB_EXPERIMENTAL` env var which defaults to false. If a platform uses an experimental feature without opting in we will fail.

Reply:
@hone

Reply:
@ekcasey how does that work with the buildpack API side of things? Is it all features if we have multiple or is it an array of features?

Reply:
@ekcasey I've updated this based on our convo last week to have a single env var set by the platform to control turning this on/off. By default we'll warn.